frontend/src/libs/components/user-profile/user-profile.tsx

```diff
@@ -21,7 +21,7 @@ const UserProfile: React.FC = () => {
     };
   });
 
-  const [activeItem, setActiveItem] = useState<string | null>(null);
+  const [activeItem, setActiveItem] = useState<string | null>('notification');
```

Comment:
```suggestion
  const [activeItem, setActiveItem] = useState<string>('notification');
```
I guess it can't be null anymore

Reply:
You were right indeed